torch/onnx/symbolic_opset12.py

```diff
@@ -21,6 +20,7 @@ def nll_loss(g, self, target, weight, reduction, ignore_index):
     reduction = sym_help._maybe_get_const(reduction, 'i')
     reduction_vals = ['none', 'mean', 'sum']
     reduction = reduction_vals[reduction]
+    return g.op("MeanSquaredDistance", input, target, reduction_s=reduction)
```

Comment:
nit: remove

Reply:
remove what?

Reply:
this is the nll_loss function, line 23 returns MeanSquaredDistance